pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "strawberry-graphql"
 packages = [ { include = "strawberry" } ]
-version = "0.126.0"
+version = "0.126.1"
```

Comment:
this is done automatically when merging 😊 can you revert it?

Reply:
sure